feature-detects/hbbtv/video.js

```diff
@@ -0,0 +1,524 @@
+/* global setTimeout */
+/*!
+{
+  "name": "HbbTV Video",
+  "property": "hbbtvvideo",
+  "tags": ["hbbtv", "oipf", "video"],
+  "async" : true,
+  "warnings": [
+    "This test is currently only compatible to HbbTV >= 1.x",
+    "All siblings of an existing HbbTV video object in the DOM will be deleted during the test"
+  ],
+  "notes": [{
+      "name": "OIPF Specification Volume 5 Release 1 v1.1 - Declarative Application Environment",
+      "href": "http://www.oipf.tv/"
+    },
+    {
+      "name": "MIT-xperts HBBTV testsuite",
+      "href": "https://github.com/mitxp/HbbTV-Testsuite"
+    }]
+}
+!*/
+/* DOC
+Tests if the play, pause and seek functionality of the HbbTV video object works correctly.
+
+At the moment the [test video](https://cdn.rawgit.com/philippsimon/Modernizr/hbbtv/feature-detects/hbbtv/test.mp4) is loaded from Github.
+For a reliable use I suggest to host the video yourself and set `window.MODERNIZR_HBBTVVIDEO_TEST_VIDEO` to that URL.
+*/
+define(['Modernizr', 'addTest', 'getBody', 'createElement', 'test/hbbtv'], function(Modernizr, addTest, getBody, createElement) {
+
+  /*
+   * Leave the console statements inside as it's the only good working way to debug the problematic HbbTV devices
```

Comment:
So I would prefer to leave the comments inside as they are the only way to debug the devices (for testing I replaced the console with a function that displays it on the device). I tested the code on around 25 TVs and half of them were extremely buggy and crashed at really weird places (worse then an old Internet Explorer ;) ).

But I could also leave them just inside of my repository and make a pull request with a clean branch.